manager.go

```diff
@@ -937,6 +940,14 @@ func (m *jobManager) resolveAsPullRequest(spec string) (*prowapiv1.Refs, error)
 
 func (m *jobManager) resolveToJob(req *JobRequest) (*Job, error) {
 	user := req.User
+	var userID string
+	if !strings.HasSuffix(req.UserProfile.Profile.Email, "@redhat.com") {
+		klog.Infof("failed to parse the UserID from the UserProfile email (not a redhat.com email)")
```

Comment:
should it be ``Debugf`` maybe? I'm not entirely familiar with the logging scheme for this component but it seems more appropriate to write external-dependent/non-critical logs with the debug level

Reply:
This was changed to `Errorf` instead.